docs/virtual_machines/creating_vms.md

```diff
@@ -0,0 +1,166 @@
+# Creating VirtualMachines
+
+The virtctl sub command `create vm` allows easy creation of virtual machine
+manifests from the command line. It leverages instance types and preferences
+and provides several flags to control details of the created virtual machine.
+
+For example there are flags to specify the name or run strategy of a
+virtual machine or flags to add volumes to a virtual machine. Instance types
+and preferences can either be specified directly or it is possible to let
+KubeVirt derive those from the volume used to boot the virtual machine.
+
+For a full set of flags and their description use the following command:
+
+```shell
+virtctl create vm -h
+```
+
+## Specifying or inferring instance types and preferences
+
+Instance types and preference can be specified with the appropriate flags, e.g.:
+
+```shell
+virtctl create vm --instancetype my-instancetype --preference my-preference
+```
+
+The type of the instance type or preference (namespaced or cluster scope)
+can be
+controlled by prefixing the instance type or preference name with the
+corresponding CRD name, e.g.:
+
+```shell
+# Using a cluster scoped instancetype and a namespaced preference
+virtctl create vm \
+  --instancetype virtualmachineclusterinstancetype/my-instancetype \
+  --preference virtualmachinepreference/my-preference
+```
+
+If a prefix was not supplied the cluster scoped resources will be used by
+default.
+
+To infer instance types and/or preferences from the volume used to boot the
```

Comment:
nit - if it lands before this link to https://github.com/kubevirt/user-guide/pull/629/commits/2ca15b6a3bfc4dfbde76b01048c76ca9b4641ceb